src/components/base/GrowScroll/index.js

```diff
@@ -33,25 +29,25 @@ class GrowScroll extends PureComponent<Props> {
     const { children, maxHeight, full, ...props } = this.props
 
     const rootStyles = {
-      overflow: 'hidden',
+      overflow: 'auto',
+      display: 'flex',
+      flexDirection: 'column',
+      flexGrow: 1,
+      flexShrink: 1,
+      position: 'relative',
       ...(full
         ? {
+            position: 'absolute',
             top: 0,
             left: 0,
             right: 0,
             bottom: 0,
           }
-        : {
-            display: 'flex',
-            flex: 1,
-            position: 'relative',
-          }),
+        : {}),
     }
 
     const scrollContainerStyles = {
-      overflowY: 'scroll',
-      marginRight: `-${80 + scrollbarWidth}px`,
-      paddingRight: `80px`,
+      overflowY: 'auto',
```

Comment:
> This can be merged without risk because previous modal has been rename LegacyModal and every components points to it.

yet this is an important change. we have to check all usage of GrowScroll right?

Reply:
yeah, we have to check tehm but I'm pretty confident. `GrowScroll` was initially meant to be fairly simple: just a growing relative box, having an absolute scrolling box inside. It has been touched multiple times because of custom scrollbars, now it returns to its original form